novawallet/Modules/Staking/NominationPools/BondMore/Base/NominationPoolBondMoreBasePresenter.swift

```diff
@@ -0,0 +1,242 @@
+    let baseInteractor: NominationPoolBondMoreBaseInteractorInputProtocol
+
+    let chainAsset: ChainAsset
+    let hintsViewModelFactory: NominationPoolsBondMoreHintsFactoryProtocol
+    let logger: LoggerProtocol
+    let balanceViewModelFactory: BalanceViewModelFactoryProtocol
+    let dataValidatorFactory: NominationPoolDataValidatorFactoryProtocol
+
+    var assetBalance: AssetBalance?
+    var poolMember: NominationPools.PoolMember?
+    var bondedPool: NominationPools.BondedPool?
+    var stakingLedger: StakingLedger?
+    var price: PriceData?
+    var fee: BigUInt?
+    var claimableRewards: BigUInt?
+    var assetBalanceExistance: AssetBalanceExistence?
+
+    init(
+        interactor: NominationPoolBondMoreBaseInteractorInputProtocol,
+        wireframe: NominationPoolBondMoreWireframeProtocol,
+        chainAsset: ChainAsset,
+        hintsViewModelFactory: NominationPoolsBondMoreHintsFactoryProtocol,
+        balanceViewModelFactory: BalanceViewModelFactoryProtocol,
+        dataValidatorFactory: NominationPoolDataValidatorFactoryProtocol,
+        localizationManager: LocalizationManagerProtocol,
+        logger: LoggerProtocol
+    ) {
+        baseInteractor = interactor
+        baseWireframe = wireframe
+        self.logger = logger
+        self.chainAsset = chainAsset
+        self.hintsViewModelFactory = hintsViewModelFactory
+        self.balanceViewModelFactory = balanceViewModelFactory
+        self.dataValidatorFactory = dataValidatorFactory
+        self.localizationManager = localizationManager
+    }
+
+    func updateView() {
+        fatalError("Must be overriden by subsclass")
+    }
+
+    func provideHints() {
+        fatalError("Must be overriden by subsclass")
+    }
+
+    func provideFee() {
+        fatalError("Must be overriden by subsclass")
+    }
+
+    func getInputAmount() -> Decimal? {
+        fatalError("Must be overriden by subsclass")
+    }
+
+    func getInputAmountInPlank() -> BigUInt? {
+        fatalError("Must be overriden by subsclass")
+    }
+
+    func getStakedAmountInPlank() -> BigUInt? {
+        guard
+            let stakingLedger = stakingLedger,
+            let bondedPool = bondedPool,
+            let poolMember = poolMember else {
+            return nil
+        }
+
+        return NominationPools.pointsToBalance(
+            for: poolMember.points,
+            totalPoints: bondedPool.points,
+            poolBalance: stakingLedger.active
+        )
+    }
+
+    func refreshFee() {
+        guard
+            let stakingLedger = stakingLedger,
+            let bondedPool = bondedPool else {
+            return
+        }
+
+        let inputAmount = getInputAmountInPlank() ?? 0
+
+        fee = nil
+
+        provideFee()
+
+        let points = NominationPools.balanceToPoints(
+            for: inputAmount,
+            totalPoints: bondedPool.points,
+            poolBalance: stakingLedger.active
+        )
+
+        baseInteractor.estimateFee(for: points)
```

Comment:
Bond more accepts amount in plank. So, no need to convert to the points. Just convert inputAmount to plank and pass to the function